webanno/webanno
# #1282 - Missing comments on export to CONLL-U

**What's in the PR**
- Temporarily backport latest DKPro Core CoNLL-U reader and writer code to enable reading/writing of sentence text comment and sentence ID comment

**How to test manually**
* Check if the export in CoNLL-U format contains the sentence text comment
* A sentence ID comment is only present if the file was originally imported from CoNLL-U and contained a sentence ID

**Automatic testing**

**Documentation**
